graspologic/inference/latent_position_test.py

```diff
@@ -1,20 +1,28 @@
 # Copyright (c) Microsoft Corporation and contributors.
 # Licensed under the MIT License.
 
+from collections import namedtuple
+
 import numpy as np
+from joblib import Parallel, delayed
 from scipy.linalg import orthogonal_procrustes
 
 from ..align import OrthogonalProcrustes
 from ..embed import AdjacencySpectralEmbed, OmnibusEmbed, select_dimension
 from ..simulations import rdpg
 from ..utils import import_graph, is_symmetric
-from collections import namedtuple
 
 lpt_result = namedtuple("lpt_result", ("p_value", "sample_T_statistic", "misc_stats"))
 
 
 def latent_position_test(
-    A1, A2, embedding="ase", n_components=None, test_case="rotation", n_bootstraps=500
+    A1,
+    A2,
+    embedding="ase",
+    n_components=None,
+    test_case="rotation",
+    n_bootstraps=500,
+    workers=1,
```

Comment:
only comment is whether to call this workers or n_jobs like sklearn/joblib. i think somewhere else in inference we call it workers to match hyppo but in autogmm it's n_jobs

Reply:
yea i was looking for `n_jobs`, but saw that LDT has `workers`. I do like n_jobs more than workers. I can change to `n_jobs` if we all agree

Reply:
ya idc too much, would probably just be good to standardize across package i.e. the other stuff in `.inference`
@alyakin314 thoughts?

Reply:
i vote revisiting this in https://github.com/microsoft/graspologic/issues/755 and just sticking with workers for now for consistency's sake

Reply:
That sounds good